packages/adconnectdump.vm/adconnectdump.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>adconnectdump.vm</id>
+    <version>0.0.0.20230320</version>
+    <authors>fox-it</authors>
+    <description>This toolkit offers several ways to extract and decrypt stored Azure AD and Active Directory credentials from Azure AD Connect servers.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="python2" />
```

Comment:
Does this work with Python3? We're thinking about removing Py2 (see #301).

Reply:
It does not! :(

Reply:
I think this is ok for tools that don't work on python 3.

Reply:
I have just realised that there is an issue with this as it mess up the path: https://github.com/mandiant/VM-Packages/issues/337